examples/dreambooth/requirements.txt

```diff
@@ -4,3 +4,4 @@ transformers>=4.25.1
 ftfy
 tensorboard
 Jinja2
+peft
```

Comment:
Should we specify a minimum version here?

Reply:
Yeah definitely, just added it!